.github/workflows/synk.yaml

```diff
@@ -0,0 +1,25 @@
+name: Snyk Rook
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
+    branches:
+      - master
+      - release-*
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: snyk/actions/setup@0.3.0
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Snyk monitor
+        run: snyk test
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
@travisn for the token https://github.com/snyk/actions#getting-your-snyk-token

Reply:
I think we are waiting for CNCF

> looks like we need to run `snyk auth` before running the test

yeah, we are waiting on CNCF side to provide this token

Reply:
Updated with the CNCF invit